frontend/src/use-cases/users/screens/show-user-details/ShowUserDetails.screen.translations.json

```
@@ -17,5 +17,9 @@
     "WouldYouLikeToDelete": [
         "Would you like to delete the user",
         "Vill du radera användaren"
+    ],
+    "DeleteAdmin": [
+        "Are you 100% sure that you want to delete the admin user? It would break everything! ",
```

Fixa översättningen på svenska sedan
```suggestion
        "Are you sure that you want to delete an admin account? ",
```